frontend/src/components/Validations/Validation.tsx

```diff
@@ -60,7 +62,7 @@ export const Validation: React.FC<ValidationProps> = (props: ValidationProps) =>
 
   // Set styles
   const textStyle = props.messageColor ? severityColor : {};
-  const iconStyle = kialiStyle(severityColor);
+  const iconStyle = classes(kialiStyle(severityColor), validation.className);
```

Comment:
@ferhoyos it looks like this keeps the `kialiStyle` prefix as the first classname since it is the first argument to `classes` so I think this will still work with OSSMC?

Reply:
good question @nrfox. Two different things to comment here:
- classes is just a util method to concatenate different CSS classes with comma, but they are still independent. That means `validation.className` does not follow OSSMC recommendation to add prefix (this won't make OSSMC to fail, the prefix is just to avoid a possible conflict with Openshift or another plugin that could have same CSS name).
- I think that you have added classname only to be able to filter the element in the test, which is not a good practice. `IconProps` has `dataTest` property, so I recommend you to replace className by dataTest (check `IconProps` interface in `KialiIcon`)

```
dataTest: 'icon-correct-validation'

.find('span[data-test*="icon-correct-validation"]')
```

Reply:
Not sure how I missed the `dataTest` prop that was already there! I'll just use that, thanks.